src/env.h

```diff
@@ -239,6 +241,7 @@ namespace node {
   V(unknown_string, "<unknown>")                                              \
   V(user_string, "user")                                                      \
   V(username_string, "username")                                              \
+  V(usingicudatadir_string, "usingICUDataDir")                                \
```

Comment:
If you're only using these strings once, there is not much point in adding them to the per-isolate cache.